include/private/rdymeshimpl.h

```diff
@@ -195,7 +195,12 @@ typedef struct RDyMesh {
   // the maximum closure size for any cell (locally stored?)
   PetscInt maxClosureSize;
 
-  Vec coords_nat, cell_conn;
+  // for output: coordinates of vertices (in vertex natural order)
```

Comment:
Do you think it would make sense to group these Vecs into an anonymous struct called `output` that makes clear wherever they're used that they're intended for output? For example:

```c
  struct {
    // ...
    Vec coords_nat;
    // ...
    Vec cell_conn;
    // ...
    Vec xc, yc, zc;
  } output;
```

Then you'd refer to these within an `RDyMesh` variable `mesh` as e.g. `mesh.output.coords_nat`, `mesh.output.xc`, and so on.

Reply:
To be clear, this is just something to consider. I don't think it's strictly necessary.

Reply:
It is a good idea to use a struct to consolidate the information. I will make the relevant change.